source/blog/2016-11-02-central-libthrift.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Farewell, maven.twttr.com!
+published: true
+post_author:
+  display_name: Bryce Anderson
+  twitter: banderson
+tags: Finagle, Finatra, Util
+---
+
+Historically Finagle has depended on org.apache.thrift libthrift
+verison 0.5.x, which happens to not be published to the Central
+Repository. We have published the artifact to the maven.twttr.com,
+but this requires users to add the maven.twttr.com repository as a
+resolver for their project. This is normally not a serious problem for
+most users, but historically maven.twttr.com has has a propensity to
+be unreliable and inaccessible from certain locations. Our long term
+goal is to transition to a current version of Apache libthrift which is
+hosted in the Central Repository, but there is a significant amount of
```

Comment:
I believe org.apache.thrift:libthrift is actually hosted by the Central Repository. Am I mistaken?